app/views/landing.html

```diff
@@ -1,5 +1,39 @@
-<div class="jumbotron">
-  <h1>Welcome!</h1>
-  <p>Your complete event solution!</p>
-  <p><a class="btn btn-primary btn-lg" role="button" href="#/eventDashboard/">My Dashboard ></a></p>
-</div>
\ No newline at end of file
+<div class="landing-jumbo">
+  <div class="row container">
+    <div class="col-xs-12">
+      <p class="text-center">
+        <i class="fa fa-users"></i>
+      </p>
+      <div class="text-center">
+        <span class="welcome">Welcome</span>
+        <div class="row">
+          <div class="col-md-6">
+            <h3>Register for an event</h3>
+            <div>
+              <span class="pull-right" ng-show="loading"><i class="fa fa-spinner fa-spin"></i> Searching...</span>
+              <p class="text-left">Enter the name of the event that you would like to register for.</p>
+              <input type="text" ng-model="staffSelectTypeahead" class="form-control"
+                     typeahead="e.name for e in eventSearch($viewValue)"
+                     typeahead-loading="loading"
+                     typeahead-wait-ms="300"
+                     typeahead-input-formatter="null"
+                     typeahead-on-select="selectEvent($item, $model, $label)">
+            </div>
+          </div>
+          <div class="col-md-6">
+            <h3>Create or manage an event</h3>
+            <div ng-show="!userIsLoggedIn">
+              <p>View registrations, manage scholarships, change questions and more.</p>
+              <p>How would you like to login?</p>
```

Comment:
3 lines of text here seems like too much and pushes the buttons down. Instead of "Create or manage an event" I might say "Login to create or manage an event" and remove the `<p>` about logging in